helm/g2p-sandbox/values.yaml

```diff
@@ -368,6 +368,7 @@ ph-ee-engine:
     tenant: "rhino,gorilla"
     hostname: bulk-connector.sandbox.mifos.io
     aws:
+      host: "http://minio:9000"
```

Comment:
can we use  AWS variables from global params so that they have to be configured in only one place
this is for the variables other than credentials configured in secret.

Reply:
please add the bucket name to the global params as well

Reply:
done.